### PR TITLE
Add "fry take my money" as a template

### PR DESCRIPTION
**Pull request by GusCaplan**
Just adding some money taking goodies.

### CODE REVIEW

File: data/templates/fry-money/config.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Shut up and take my money
+link: http://knowyourmeme.com/memes/fry-take-my-money
+default:
+- SHUT UP AND
+- TAKE MY MONEY!
+aliases:
+- frymoney
+- shut-up-and-take-my-money
+- take-my-money
+regexes:
```
Comment (jacebrowning) [inline data/templates/fry-money/config.yml]: I think the follow should all return matches:
-  `/magic/shut up and take my money`
-  `/magic/shut up and do that thing`
- `/magic/stop talking and take my money`

**Comment by GusCaplan** [comment]
@jacebrowning what is /magic?

**Comment by jacebrowning** [comment]
It's not yet exposed as a feature in the UI, but http://memegen.link/magic is the API route that uses the regular expressions to "guess" what meme you are trying to make. You can tack strings on to the end of that route to get a list of matches.

**Comment by jacebrowning** [comment]
Independent of this PR, I'll look at creating a better testbed for this pattern matching.

**Comment by GusCaplan** [comment]
got it

File: data/templates/fry-money/config.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- SHUT UP AND
+- TAKE MY MONEY!
+aliases:
+- frymoney
+- shut-up-and-take-my-money
+- take-my-money
+- shut-up-and-do-that-thing
+- stop-talking-and-take-my-money
+regexes:
+- (.+)/?(shut up and take my money)
```
Comment (jacebrowning) [inline data/templates/fry-money/config.yml]: Sorry, I didn't mean literally use those phrases, just that those types of phrases should also match.

**Comment by GusCaplan** [comment]
@jacebrowning i added the regex. Isn't that what you wanted?

**Comment by jacebrowning** [comment]
I'm going to work on making the instructions more clear. I've manually pulled in these changes.

**Pull request closed without merging**